File: ProjetoIgreja/Back/src/IgrejaJdLilah.Domain/Entidades/Palestrante.cs
using System;
using System.Collections.Generic;

namespace IgrejaJdLilah.Domain.Entidades
{
    public class Palestrante
    {
       public int Id { get; set; }  
       public string Nome { get; set; } 
       public string MiniCurriculo { get; set; }      
       public int ImagemUrl { get; set; }
       public string Telefone { get; set; }
       public string Email { get; set; }
       public string Denominacao { get; set; }
       public DateTime DataCadastro { get; set; }
       public bool FazParteMesmaDenominacao { get; set; }
       public int? EnderecoId { get; set; }
       public Endereco Endereco { get; set; }
       public IEnumerable<RedeSocial> RedeSociais { get; set; }
       public IEnumerable<PalestranteEvento> PalestranteEventos { get; set; }
    }
}

File: ProjetoIgreja/Front/IgrejaJdLilah-APP/src/app/models/Palestrante.ts
import { Endereco } from "./Endereco";
import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial";


export interface Palestrante {
  id:  number;
  nome: string;
  miniCurriculo: number;
  imagemUrl: string;
  telefone: string;
  email: string;
  denominacao: string;
  dataCadastro: Date;
  fazParteMesmaDenominacao: boolean;
  enderecoId?: number;
  endereco: Endereco[];
  redeSociais: RedeSocial[];
  palestranteEvento: Evento[];
}

